tensorflow/compiler/tf2tensorrt/convert/convert_nodes.cc

```diff
@@ -2185,40 +2185,24 @@ Status ConvertExpandDims(OpConverterParams* params) {
   TRT_TensorOrWeights input_tensor = inputs.at(0);
   const nvinfer1::Dims dims = input_tensor.GetTrtDims();
   std::vector<int> input_dims(dims.d, dims.d + dims.nbDims);
-  // Add batch dim back.
-  input_dims.insert(input_dims.begin(), -1);
-  const int input_rank = input_dims.size();
   // Get axis to expand on.
-  TRT_ShapedWeights weights = inputs.at(1).weights();
-  if (weights.count() != 1) {
+  auto axis = inputs.at(1).weights().GetSpan<int>();
```

Comment:
could this be const auto ? or is this modified?

Reply:
‘GetSpan<T>‘ returns a ‘Span<const T>‘, I think that already means it can’t be modified? 

This variable is not modified so it should be const